csFastFloat/Constants/Utf8Literals.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+namespace csFastFloat
+{
+  internal partial class Utf8Literals
+  {
+    [StringLiteral.Utf8("infinity")]
```

Comment:
This way it's much nicer 😄 
If you see in Visual Studio errors regarding these definitions, please re-start Visual Studio.
The editor needs to hook up the [source generator](https://devblogs.microsoft.com/dotnet/introducing-c-source-generators/).